hr_attendance_management/__manifest__.py

```diff
@@ -63,12 +63,12 @@
         'views/hr_change_day_wizard_view.xml',
         'views/hr_change_day_requests.xml'
     ],
-    'demo': [
-        'demo/hr_config.xml',
-        'demo/resource_calendar.xml',
-        'demo/hr_attendance_day.xml',
-        'demo/hr_contract.xml',
-    ],
+    # 'demo': [
```

Comment:
I suppose you had troubles with demo data, but please uncomment this. It should work with it (tests also need them I guess).